Makefile

```diff
@@ -195,15 +195,21 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
-test: install-unittest-plugin manifests generate fmt vet lint get-go-junit-report
+ENVTEST_ASSETS_BIN_DIR=$(ENVTEST_ASSETS_DIR)/tmp
+export KUBEBUILDER_ASSETS=$(ENVTEST_ASSETS_DIR)/bin
+test: install-unittest-plugin manifests generate fmt vet lint get-go-junit-report setup-envtest
 	helm unittest --helm3 --output-type JUnit --output-file $(TMPDIR)/unit-tests.xml helm-charts/verticadb-operator
 	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.2/hack/setup-envtest.sh
+	mkdir -p ${ENVTEST_ASSETS_BIN_DIR}
+	mkdir -p ${KUBEBUILDER_ASSETS}
+	$(SETUP_ENVTEST) use --bin-dir $(ENVTEST_ASSETS_BIN_DIR)
+	find $(ENVTEST_ASSETS_BIN_DIR)/ -exec cp {} $(KUBEBUILDER_ASSETS)/ \;
+	sudo rm -r $(ENVTEST_ASSETS_BIN_DIR)
```

Comment:
I do not like this sudo. First I did not intend to add it but the github checks were failing. Using setup-envtest, the binaries are generated in <your-dir>/k8s/<version>. So I generated them in testbin/tmp/k8s/<version> them copy them to testbin/bin to then delete testbin/tmp.